src/components/header/Header.jsx

```diff
@@ -56,17 +60,23 @@ export function Header() {
         </div>
 
         <ul onClick={toggleSideMenu}>
-          <li className="my-sm">
-            <Link to="/login">
-              <button className="btn">Login</button>
-            </Link>
-          </li>
+          {token ? (
+            <li className="my-sm">Hi, {userName} </li>
+          ) : (
+            <>
+              <li className="my-sm">
+                <Link to="/login">
+                  <button className="btn">Login</button>
+                </Link>
+              </li>
 
-          <li className="my-sm">
-            <Link to="/signup">
-              <button className="btn">Singup</button>
-            </Link>
-          </li>
+              <li className="my-sm">
+                <Link to="/signup">
+                  <button className="btn">Singup</button>
```

Comment:
Sign up typo

Reply:
Thanking for pointing that out, Kaustubh.